src/v2/view-builder/views/ov/ChallengeOktaVerifyFastPassView.js

```diff
@@ -90,10 +90,17 @@ const Body = BaseFormWithPolling.extend(Object.assign(
         Logger.error(`Something unexpected happened while we were checking port ${currentPort}.`);
       };
 
+
+      const cancelPolling = () => {
+        this.options.appState.trigger('invokeAction', 'currentAuthenticator-cancel');
+      };
+
       const doProbing = () => {
         return checkPort()
           // TODO: can we use standard ES6 promise methods, then/catch?
-          .done(onPortFound)
+          .done(() => {
```

Comment:
possible to address that TODO ? in this pr

Reply:
This is out of scope. I will look into it while working on refactoring.